### PR TITLE
Removed commented out code from std.stdio

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/stdio.d [hunk 1]
```diff
@@ -3297,15 +3291,7 @@ unittest
     f.close();
     scope(exit) { std.file.remove(deleteme); }
     assert(cast(char[]) std.file.read(deleteme) == "Hello, world number 42!");
-    // // test write on stdout
-    //auto saveStdout = stdout;
-    //scope(exit) stdout = saveStdout;
-    //stdout.open(file, "w");
     Object obj;
```
Comment (burner) [inline std/stdio.d]: while you're at it, could the Object  obj be removed?

**Comment by burner** [comment]
LGTM

**Comment by JackStouffer** [comment]
@burner fixed

**Comment by burner** [comment]
Auto-merge toggled on

**Pull request merged**